### ISSUE TITLE
Add aria-label attribute to IconButtons

### ISSUE THREAD

**Issue by shayc** [cboard-org/cboard#111]
Some of the `IconButton` components are missing [aria-label attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute):

- [x] `FullScreenDialog.js`
- [x] `SymbolOutput.js`

**Comment by Luftzig** [comment]
Taken

**Comment by shayc** [comment]
thanks!
